_sass/prism.scss

```diff
@@ -21,7 +21,7 @@ pre[class*="language-"] {
   hyphens: none;
 }
 pre[class*="language-"] {
-  padding: 1em;
+  padding: 1.8em 1em 1em 1em;
```

Comment:
Just curious, what was this change for 🤔

Reply:
The additional `padding-top` was to prevent cutting off any first-line text.
![padding](https://user-images.githubusercontent.com/6802326/226050428-8eb0b4af-790f-4afe-956d-de735b8c3c1f.png)